test/download-iso

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -xeu
+
+TEST_OS=$1
+TEST_COMPOSE=$2
```

Comment:
Please quote. Test composes shouldn't have spaces in them, but it's a good habit.